shell/src/main/java/alluxio/cli/fs/command/RmCommand.java

```diff
@@ -70,6 +70,14 @@ public final class RmCommand extends AbstractFileSystemCommand {
           .hasArg(false)
           .desc("remove mount points in the directory")
           .build();
+  private static final Option SYNC_DIR_NEXT_TIME =
+      Option.builder()
+          .longOpt("syncDirNextTime")
+          .required(false)
+          .hasArg(false)
+          .desc("Marks a directory to either trigger a metadata sync or skip the "
+              + "metadata sync on next access.")
```

Comment:
I don't remember now, is it `--syncDirNextTime true/false` or just `--syncDirNextTime` without input? If no input (having this arg always means `true`), then we need to update the comment here:
```
          .longOpt("skipSyncNextTime")
          .required(false)
          .hasArg(false)
          .desc("Marks a directory to skip the metadata sync on next access.")
```

Reply:
Thanks for the suggestion. The default behavior without any option for `rm` means rm a file or directory and do not reset the directChildrenState of its parent dir to false, so I don't quite sure whether it should be named `skipSyncNextTime` which is a antonym of `syncDirNextTime`